docs/source/tutorials/portfolio_optimization.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>typo: 'brutal force' -&gt; 'brute force'</p><p> </p><p>Might be good to remind the reader about what the cost function is, e.g. "We first use brute force search to calculate the cost $x^TQx$ for each $x\in\{0,1\}^n$, where $n$ is the number of stocks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>